pythran/pythonic/include/__builtin__/enumerate.hpp

```diff
@@ -14,23 +14,19 @@ namespace pythonic
 
     namespace details
     {
-      // FIXME return value may be a type::make_tuple
```

Comment:
I thought the comment meant: beware, the return value may be different
from the type described by the  iterator trait. I fixed this somehow and
thought it was ok to remove the comment.